### PR TITLE
Add findWindows method

### LINKED ISSUE #1: Support to covert back from IANA to windows
I believe it would be nice to add support to add support to covert timezones the other way. I noticed some other libs that do it and if I'm not wrong one of them even forked from this repo. But I believe it would be nicer to keep it altogether in this lib. I already have the code for a pull request (with tests and lint passing), but wasn't able to send it to the repo. Let me know if you want me to send it!

### PR DESCRIPTION
**Pull request by willyboy**
This adds a method that allows you to look up the windows timezones when you have an iana timezone.

Fixes #1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "windows-iana",
-  "version": "0.0.0-development",
-  "description": "A small tool to convert Windows time zones to IANA",
+  "version": "0.0.1-development",
```
Comment (rubenillodo) [inline package.json]:
```suggestion
  "version": "0.0.0-development",
```

No need to change this. The project uses [semantic-release](https://github.com/semantic-release/semantic-release).

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+  "description": "A small tool to convert Windows time zones to IANA and vice versa",
```
Comment (rubenillodo) [inline package.json]:
```suggestion
  "description": "A small tool to convert between Windows time zones and IANA",
```

**Comment by willyboy** [comment]
Done! Sorry for the delay. Looks like one of the commits failed because it was autocancelled but the previous CI check succeeded so not worried about it.

**Comment by willyboy** [comment]
Oops. Clicked the wrong button.

**Pull request merged**